capstone/WEB-INF/datastore-indexes.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file shouldn't be necessary since it didn't actually help us :(

Reply:
Yeah, my bad. Forgot to update this after you figured out the issue!